src/celengine/overlay.cpp

```diff
@@ -11,25 +11,28 @@
 #include <cstring>
 #include <Eigen/Core>
 #include <celmath/geomutil.h>
-#include <celttf/truetypefont.h>
 #include <celutil/color.h>
 #include "overlay.h"
 #include "rectangle.h"
 #include "render.h"
-#include "texture.h"
 #include "vecgl.h"
+#include "textlayout.h"
 
 using namespace std;
 using namespace Eigen;
 using namespace celmath;
+using namespace celestia::engine;
 
 Overlay::Overlay(Renderer& r) :
     renderer(r)
 {
+    layout = make_unique<TextLayout>(r.getScreenDpi());
```

Comment:
could you address this warning: 
Variable 'layout' is assigned in constructor body. Consider performing initialization in initialization list.

Reply:
fixed